bin/sce_qc_report.R

```diff
@@ -140,16 +146,20 @@ processed_sce_meta <- metadata(processed_sce)
 sample_ids <- unlist(stringr::str_split(opt$sample_id, ",|;")) |> sort()
 
 # check for multiplexing
-multiplexed = if (length(sample_ids) > 1){TRUE} else {FALSE}
+multiplexed <- if (length(sample_ids) > 1) {
+  TRUE
+} else {
+  FALSE
+}
```

Comment:
I don't know if this is updated in the main branch, but it should be!

```suggestion
multiplexed <- length(sample_ids) > 1
```